CHANGELOG.MD

```diff
@@ -1,6 +1,20 @@
 # Change Log
+
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](https://semver.org/).
 
+
+## v1.0.1 - 2023-10-18
+
+> **Feature**: Update sample .env files
```

Comment:
Do these updates have ticket numbers? I'm not sure if we're still doing this, but we used to add updates in this format:

* Added option to add a custom link for the Hub Update custom post type. ([DESENG-410](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-410))

Reply:
You're right; I linked the relevant ticket in [61145ce](https://github.com/bcgov/met-public/pull/2320/commits/61145cefa7c6850b1e1336dc59f0324f096e4f7c). Thanks!